Simplenote/SPAppDelegate.m

```diff
@@ -7,6 +7,7 @@
 //
 
 #import "SPAppDelegate.h"
+#import <Contacts/Contacts.h>
```

Comment:
Kind of a nitpick, but Apple recommend using the module-based `@import` for system frameworks these days.